src/swap/components/SwapProvider.tsx

```diff
@@ -313,7 +321,9 @@ export function SwapProvider({
     handleSubmit,
     lifeCycleStatus,
     isTransactionPending,
+    maxSlippage,
     setLifeCycleStatus,
+    setMaxSlippage,
```

Comment:
As mentioned yesterday, we are moving away from a pattern of making the Context bigger, and instead evreywhing has to be controlled between the Lifecycle Status.